.azure-pipelines/testscripts_analyse/analyse_testscripts.py

```diff
@@ -165,9 +173,12 @@ def check_PRChecker_coverd(test_scripts, topology_type_pr_test_scripts):
             expanded_test_scripts.append(test_script)
 
     # Check if a script is included in the PR checker for the corresponding topology type
+    # And if this script is skipped in PR checker
     for test_script in expanded_test_scripts:
         topology_type = topo_name_to_type(test_script["topology"])
 
+        test_script["skipped"] = test_script["testscript"] in topology_type_pr_test_skip_scripts.get(topology_type, "")
```

Comment:
The comment at the top of the file shows the sample return value of this script can include now `skipped` as well.

Reply:
Good catch! Let me add it.